packages/grafana-ui/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -189,7 +189,7 @@ export const ContextMenu: React.FC<ContextMenuProps> = React.memo(({ x, y, onClo
           renderItem={(item, index) => {
             return (
               <>
-                <ContextMenuGroup group={item} onItemClick={onClose} />
+                <ContextMenuGroup group={item} onClick={onClose} />
```

Comment:
changed to "onClick" because we were confused that it was something special about the *item* not the whole group